cumulusci/tasks/dx_convert_from.py

```diff
@@ -0,0 +1,41 @@
+from pathlib import Path
+
+from cumulusci.tasks.sfdx import SFDXBaseTask
+
+
+class DxConvertFrom(SFDXBaseTask):
+    task_options = {
+        "command": {
+            "description": "The full command to run with the sfdx cli.",
+            "required": True,
+        },
```

Comment:
I think we may as well remove the `command` option since we're making another backwards-incompatible change anyway

Reply:
If we remove the `command` option, from `task_options`, then would we just keep the current logic in `_init_options()` but now hardcode a value for `self.options["command"] = f"force:source:convert -d {self.options['src_dir']}"` so that the parent tasks can access that value?

Reply:
Yeah, that's basically what I had in mind